docker-compose.yml

```
@@ -256,6 +256,20 @@ services:
     networks:
       - kafka-network
 
+  zipkin:
+    image: openzipkin/zipkin:3
+    container_name: antcamp-zipkin
+    ports:
+      - "${ZIPKIN_PORT}:9411"
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:9411/health" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)